src/Host/Broker/Impl/Sessions/Session.cs

```diff
@@ -160,6 +161,18 @@ public class Session {
             HostToClientWorker(_process.StandardOutput.BaseStream, hostEnd).DoNotWait();
         }
 
+        private void StartSession() {
+            _process.Start();
+            _process.WaitForExit(250);
+            if (_process.HasExited && _process.ExitCode != 0) {
+                var message = ErrorCodeConverter.MessageFromErrorCode(_process.ExitCode);
```

Comment:
Here's something that I found:

> By convention, processes that exit cleanly return either 0, 1, or a Win32 error code (which are all small positive numbers) so an exit code with the top bit set usually means a status code, typically because of an unhandled exception.

Also, in ntstatus.h, you can see that the "error" category starts with `0xC0000001L` (`STATUS_UNSUCCESSFUL`). So I think it's safe to ignore everything that's less than that.